civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6191,6 +6191,34 @@ function _get_task_filter_by_date($date, $normalize = false) {
   return $normalize ? ($filter == 'tomorrow' ? 'week' : $filter) : $filter;
 }
 
+/**
+ * Sets the maximum viewport scale to 1 to prevent auto zooming on input focus
+ * @see https://stackoverflow.com/questions/11064237/prevent-iphone-from-zooming-form
+ *
+ * @param object $head_elements
+ */
+function _set_maximum_scale_to_viewport_meta_tag(&$head_elements) {
+  foreach ($head_elements as $key => $tag) {
+    if (isset($tag['#attributes']['name']) && $tag['#attributes']['name'] === 'viewport') {
```

Comment:
What if we just do `$tag['#attributes']['name'] === 'viewport'`

Reply:
@deb1990 this will throw a warning if the key does not exist. This was also recommended by a community https://stackoverflow.com/questions/49656547/how-to-override-already-added-meta-tags-in-drupal-7/49656741#49656741 and official pages https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_html_head_alter/7.x